Lab 1 (1).html

```diff
@@ -13104,7 +13104,7 @@
 </div><div class="inner_cell">
 <div class="text_cell_render border-box-sizing rendered_html">
 <p>Normal Distribution!</p>
-<p>A normal distribution is characterized by a bell shaped "gaussian" curve, giving the probability density based on a defined sigma and mean value.</p>
+<p>A normal distribution is characterized by a bell shaped "gaussian" curve, giving the probability density based on a defined sigma and mean value. Potentially add a bit more detail about gaussian distributions, maybe applications.</p>
```

Comment:
Just thought it might be good to have a bit more background about the gaussian here or maybe applications. Although that stuff really isn't relevant to this lab report so maybe not, just an idea.